tests/python_functional/functional_tests/applications/test_applications_with_app_parser.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I would create a `parsers/app-parser` directory instead of `applications`

Reply:
Maybe it will be more readable (searchable) if the testcase can be separate into two files: topic-syslog-raw and topic-syslog

Reply:
Fixed